collectd-kong/README.md

```diff
@@ -0,0 +1,149 @@
+# ![](./img/integrations_kong.png) Kong
+
+Metadata associated with the Kong plugin for collectd can be found
+[here](https://github.com/signalfx/integrations/tree/master/collectd-kong).
+The relevant code for the integration can be found [here](https://github.com/signalfx/collectd-kong) and for its
+Kong plugin dependency [here](https://github.com/signalfx/kong-plugin-signalfx).
+
+- [Description](#description)
+- [Features](#features)
+- [Requirements and Dependencies](#requirements-and-dependencies)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+This is the SignalFx Kong collectd plugin to provide users with the ability to gather and report their service traffic
+metrics with collectd, in tandem with [kong-plugin-signalfx](https://github.com/signalfx/kong-plugin-signalfx).
+
+This plugin emits metrics for configurable request/response lifecycle groups including:
+
+* Counters for response counts
+* Counters for cumulative response and request sizes
+* Counters for cumulative request, upstream, and kong latencies
+
+These request/response lifecycle groups can be optionally partitioned by tunable levels of granularity by:
+
+* API or Service Name/ID
+* Route ID
+* Request HTTP Method
+* Response HTTP Status Code
+
+In addition to these groups, system-wide connection stats can be provided, including:
+
+* A counter for total fielded requests
+* Gauges for active connections and their various states
+* A gauge for database connectivity
+
+#### FEATURES
+
+##### Built-in dashboards
+
+- **Kong Servers Overview**: High-level Kong performance for all reporting server
+
+  [<img src='./img/kong_servers_overview.png' width=200px>](./img/kong_servers_overview.png)
+
+- **Kong Server Overview**: High-level Kong performance for individual server
+
+  [<img src='./img/kong_server_overview.png' width=200px>](./img/kong_server_overview.png)
+
+- **API Objects**: API object activity metrics
+
+  [<img src='./img/api_objects.png' width=200px>](./img/api_objects.png)
+
+- **Service Objects**: Service object activity metrics
+
+  [<img src='./img/service_objects.png' width=200px>](./img/service_objects.png)
+
+- **Route Objects**: Route object activity metrics
+
+  [<img src='./img/route_objects.png' width=200px>](./img/route_objects.png)
+
+### REQUIREMENTS AND DEPENDENCIES
+
+This plugin requires:
+
+| Software          | Version        |
+|-------------------|----------------|
+| collectd | 4.9+ |
+| Python plugin for collectd | (included with [SignalFx collectd agent](https://github.com/signalfx/integrations/tree/master/collectd)[](sfx_link:sfxcollectd)) |
```

Comment:
Below you say you can install this plugin even if you are using the Smart Agent. So these requirements don't seem exactly accurate. If there are two different sets of requirements and install instructions, it should be easier for users to determine which is which and what they should do.